src/pages/cms-information-page/index.tsx

```diff
@@ -127,32 +167,7 @@ const InformationPage: FC<Props> = () => {
           <SC.SideMenu isSticky={isSticky} menuItems={menuItems} />
           {navOpen && <SC.SideMenuSmall menuItems={menuItems} />}
         </SC.Aside>
-        <SC.Article>
-          <SC.Title>{title}</SC.Title>
-          <SC.Description>{subtitle}</SC.Description>
-          {Content?.map(
-            component =>
-              (isBasicParagraph(component) && (
-                <SC.Content>
-                  <Markdown allowHtml>{component?.Content ?? ''}</Markdown>
-                </SC.Content>
-              )) ||
-              (isBasicImage(component) && (
-                <SC.ImageWrapper key={component?.id}>
-                  <SC.Image
-                    alt={`${component?.media?.[0]?.alternativeText}`}
-                    src={`${FDK_CMS_BASE_URI}${component?.media?.[0]?.url}`}
-                  />
-                  {component?.media?.[0]?.caption && (
-                    <SC.ImageText>
-                      {localization.informationPage.imageText}
-                      {component?.media?.[0]?.caption}
-                    </SC.ImageText>
-                  )}
-                </SC.ImageWrapper>
-              ))
-          )}
-        </SC.Article>
+        {!loading && getPage()}
```

Comment:
nå vil loading bare være en blank side? Kanskje bedre å feks legge til Spinner?
```suggestion
        {getPage()}
```